### PR TITLE
Feature/bimdata slider

### PR DESCRIPTION
**Pull request by LrxGaelle**
- [ ] I have tested my component
- [ ] I have updated the documentation or there is no documentation needed

## Libraries that use the DS (need to merge first)

- [ ] [BCF Components](https://github.com/bimdata/bcf-components)
- [ ] [BIMData Components](https://github.com/bimdata/bimdata-components)
- [ ] [Building maker](https://github.com/bimdata/building-maker)

## Repos that use the DS (and that I need to check after libraries have been merged)

- [ ] [Viewer](https://github.com/bimdata/viewer)
- [ ] [Platform](https://github.com/bimdata/platform)
- [ ] [SDK](https://github.com/bimdata/bimdata-viewer-sdk)
- [ ] [Marketplace](https://github.com/bimdata/marketplace-front)
- [ ] [Documentation](https://github.com/bimdata/documentation)

### CODE REVIEW

**Review by Amoki — APPROVED** [review]

File: src/web/views/Components/Slider/Slider.vue [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          />
+          <BIMDataInput
+            type="number"
+            placeholder="Max range"
+            v-model="maxRange"
+          />
+          <BIMDataInput
+            type="number"
+            placeholder="Slider default value"
+            v-model="slider"
```
Comment (NicolasRichel) [inline src/web/views/Components/Slider/Slider.vue:34]: You should decompse `v-model="slider"` this way ton avoid warnings in the console:
```
:modelValue="slider"
@update:modelValue="slider = +$event"
```

File: src/web/views/Components/Slider/props-data.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "Use this props to define max range.",
+    "100",
+  ],
+  [
+    "step",
+    "Number, String",
+    "",
+    "null",
+    "You can change the default step increment.",
+    "10",
```
Comment (NicolasRichel) [inline src/web/views/Components/Slider/props-data.js:33]: `step` default value is **1** not **10**.

File: src/web/views/Components/Slider/props-data.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    "",
+  ],
+  [
+    "disabled",
+    "Boolean",
+    "",
+    "false",
+    "Use this props to disable slider.",
+    "true",
+  ],
```
Comment (NicolasRichel) [inline src/web/views/Components/Slider/props-data.js:50]: The `tooltip` props is not documented here. ;-)

**Review by NicolasRichel — CHANGES_REQUESTED** [review]

File: src/web/views/Components/Slider/props-data.js (hunk 2 above)
Comment (LrxGaelle) [reply src/web/views/Components/Slider/props-data.js:33]: The number 10 here is for the "example" part

**Review by NicolasRichel — APPROVED** [review]

**Review by Kurtil — APPROVED** [review]

**Pull request merged**